react-rendering-lists/render-list-app/src/App.tsx

```diff
@@ -0,0 +1,25 @@
+import { PokemonList } from './PokemonList';
+import './App.css';
+
+export type Pokemon = {
```

Comment:
I think it's more natural to put this in PokemonList.tsx, where it's used most heavily. You could then import both PokemonList and Pokemon from there in a single import statement.